.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
```

Comment:
Because when people submit a PR they might still have lingering problems (like I did a lot of times haha), can we also add a section that allows users to specify potential issues they have or advice they'd like to seek? I know it's already a lot of documention but I definitely think it's a good idea to have one such template. At worst, people can leave it blank, and at best, this is going to facilitate and help organize contributors' description of their PR.

Reply:
Mm sounds good. I will be making amends